### ISSUE TITLE
Smaller architecture

### ISSUE THREAD

**Issue by mcandre** [RobertElderSoftware/recc#9]
Why not start the instructions for the CPU with NAND? That way, you do not have to implement so much, without sacrificing Turning completeness. Any other commands are simply conveniences to be added later.

**Comment by RobertElder** [comment]
Yes, I had thought about making it even smaller, but at a certain point the instruction becomes so small that it starts to become obfuscated.  The idea was to make a small/simple/understandable architecture.  Having said that, I do think there are still changes that should be made to the instruction set.  It has been a few years since I've worked on this project regularly, but I recall that it seemed like a good idea next to add a 'return' or 'branch to function' instruction of some sort to make it easier and less instructions to return from a function call.  Similar to how ARM does it with bl instruction.
